eng/tools/lint-diff/src/lint-diff.ts

```diff
@@ -91,32 +88,32 @@ export async function main() {
   );
   console.log(`Using @microsoft.azure/openapi-validator version: ${validatorVersion}\n`);
 
-  await runLintDiff(
-    beforeArg as string,
-    afterArg as string,
-    changedFilesPath as string,
-    outFile as string,
-    baseBranch as string,
-    compareSha as string,
-    githubRepoPath as string,
-  );
+  const beforePath = beforeArg as string;
+  const afterPath = afterArg as string;
+
+  await runLintDiff(beforePath, afterPath, outFile as string, githubRepoPath as string);
 }
 
 async function runLintDiff(
   beforePath: string,
   afterPath: string,
-  changedFilesPath: string,
   outFile: string,
-  baseBranch: string,
-  compareSha: string,
   githubRepoPath: string,
 ) {
+  const baseSha = await getSha(resolve(beforePath));
+  const changedFiles = await getChangedFiles({
+    cwd: resolve(afterPath),
+    baseCommitish: baseSha,
+    headCommitish: "",
+    paths: ["specification"],
+  });
```

Comment:
```suggestion
  const baseSha = await getSha(resolve(beforePath));
  const changedFiles = await getChangedFiles({
    cwd: resolve(afterPath),
    paths: ["specification"],
  });
```

It's safest to use the same parameters you used previously, to generate `changed-files.txt`.  If you want to add support for a new feature (like comparing uncommitted files), let's move that to a separate PR.